src/wallet/rpcwallet.cpp

```diff
@@ -1690,7 +1690,7 @@ UniValue listsinceblock(const JSONRPCRequest& request)
             "\nGet all transactions in blocks since block [blockhash], or all transactions if omitted\n"
             "\nArguments:\n"
             "1. \"blockhash\"            (string, optional) The block hash to list transactions since\n"
-            "2. target_confirmations:    (numeric, optional) The confirmations required, must be 1 or more\n"
+            "2. target_confirmations:    (numeric, optional, default=1) Return the nth block hash from the main chain. 1 would mean the bestblockhash (see lastblock in return value)\n"
```

Comment:
Maybe add a note that transactions with fewer confirmations than this are still included in the response, only lastblock is changed by this argument.